### ISSUE TITLE
Additional Columns in the table view

### ISSUE THREAD

**Issue by smza** [seek4science/seek#789]
It would be useful to have the access right information (private / public) related to the project or a datafile in the table view.

**Additional context*
Right now when you are in the project to recheck whether the file/project is public or private you need to use the manage file action. It would make it easy to have all this information when we click on the file and when we use the table view.

**Comment by stuzart** [comment]
In the default and condensed view, there is an icon on the title that represents it's privacy setting. 

This in only visible to items where you have access to this information, otherwise would be revealing confidential information.

Could also include this on the table view in a column where the information is available

**Comment by smza** [comment]
I can't see the icon that you mention (see image). I've access to the first project (Biocare)
<img width="1142" alt="Screenshot 2021-11-17 at 11 12 05" src="https://user-images.githubusercontent.com/20230281/142190511-4b356b6e-d9b7-434d-8bd3-3dafaadb740c.png">

**Comment by stuzart** [comment]
they are project, they don't have sharing permissions.

Here is an example when viewing my SOPs on the test server

![image](https://user-images.githubusercontent.com/201658/142191424-a459c366-372b-4f54-a218-17885daaff58.png)

**Comment by smza** [comment]
This is message is for people who are not a part of the project or do not have access to the project. In that case what I'm asking for is information for the owner of the project/datafile.

**Comment by alaninmcr** [comment]
I can see those icons when I have some sort of control over an asset. If it does not show the globe, does that mean it is interpreted as not public? The spanner appears to mean custom policy. I can get the lock by making it only visible to me (the owner).

**Comment by stuzart** [comment]
yes, you can only see the icons if the permissions are available to you. If you don't have access to see the permissions (i.e. manage rights) then it isn't available to you. Don't so could leak confidential information about how the owner shared the item.

**Comment by smza** [comment]
I have a request from user that there should be some way to know that the file/project is confidential. We can see this information in the asset report but its very hidden hence not useful. I think indicating the file is not accessible to everyone doesn't give away any information. In the tree-view also we give this information "these files are hidden or something".

**Comment by stuzart** [comment]
An Icon or field to say that something is publicly accessible I don't think would be a problem. You can always see this by looking at what is available when logged out.
I'm still not sure what you mean by a confidential project though. Do you mean one you are not a member of ?

**Comment by smza** [comment]
All I want is an icon on the projects that are not accessible to public. Like the lock icon you have.
